### PR TITLE
Adjust Contact Info, Update App Icon + Accent Color

### PR DESCRIPTION
**Pull request by bryant-jimenez**
# Update Contact Info + App Icon

## :recycle: Current situation & Problem
- The application required an app icon update, accent color update to align with app icon color scheme, and contact information update for the project contacts.

## :gear: Release Notes 
- No new features added, only visual changes. Contact view information for three main contacts changed to only allow for email contact and website viewing.

## :books: Documentation
- No need to update documentation, only smaller visual changes

## :white_check_mark: Testing
- Updated text within prior working version of app, visual changes

## :pencil: Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: Behavior/Contacts/Contacts.swift
Comment (aydinzahedi) [inline Behavior/Contacts/Contacts.swift:1]: Awesome work using the string catalog for the bio!

**Review by aydinzahedi — APPROVED** [review]
Great job! Especially appreciated that you used the PR template well to document, and great job using the string catalog for the bios.

File: Behavior/Resources/Localizable.xcstrings [hunk 1]
```diff
@@ -134,6 +134,17 @@
         }
       }
     },
+    "EMMA_BRUNSKILL_BIO" : {
+      "extractionState" : "manual",
```
Comment (philippzagar) [inline Behavior/Resources/Localizable.xcstrings]: Please fix the `manual` extraction state, Xcode automatically synthesizes the String catalog upon every build, meaning you don't need to declare the string keys yourself

Comment (bryant-jimenez) [reply Behavior/Resources/Localizable.xcstrings]: Thanks for the review Phillip! I've gone ahead and changed the extraction state. Feel free to let me know if there's anything else.

**Comment by philippzagar** [comment]
@bryant-jimenez Please make the changes and then request a review again! :)

**Pull request merged**